src/JSInterop/Microsoft.JSInterop/ref/Microsoft.JSInterop.netstandard2.0.cs

```diff
@@ -12,21 +12,32 @@ public static partial class DotNetDispatcher
         [Microsoft.JSInterop.JSInvokableAttribute("DotNetDispatcher.ReleaseDotNetObject")]
         public static void ReleaseDotNetObject(long dotNetObjectId) { }
     }
-    public partial class DotNetObjectRef : System.IDisposable
+    public static partial class DotNetObjectRef
     {
-        public DotNetObjectRef(object value) { }
-        public object Value { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
+        public static Microsoft.JSInterop.DotNetObjectRef<TValue> Create<TValue>(TValue value) where TValue : class { throw null; }
+    }
+    public sealed partial class DotNetObjectRef<TValue> : System.IDisposable where TValue : class
+    {
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("This API is meant for JSInterop infrastructure and should not be used by user code.")]
+        public DotNetObjectRef() { }
+        public DotNetObjectRef(TValue value) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("This API is meant for JSInterop infrastructure and should not be used by user code.")]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("__dotNetObject")]
+        public long TrackingId { get { throw null; } set { } }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute]
+        public TValue Value { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public void Dispose() { }
-        public void EnsureAttachedToJsRuntime(Microsoft.JSInterop.IJSRuntime runtime) { }
+        public static implicit operator TValue (Microsoft.JSInterop.DotNetObjectRef<TValue> result) { throw null; }
     }
     public partial interface IJSInProcessRuntime : Microsoft.JSInterop.IJSRuntime
     {
         T Invoke<T>(string identifier, params object[] args);
     }
     public partial interface IJSRuntime
     {
-        System.Threading.Tasks.Task<T> InvokeAsync<T>(string identifier, params object[] args);
-        void UntrackObjectRef(Microsoft.JSInterop.DotNetObjectRef dotNetObjectRef);
```

Comment:
why does this go away?

Reply:
See https://github.com/aspnet/Extensions/pull/1704#discussion_r283906655. We don't need this API to implement disposing the ref. Users can dispose the actual `DotNetObjectRef<T>` instance which should stop tracking it.